crates/edr_napi/package.json

```diff
@@ -23,8 +23,10 @@
   "license": "MIT",
   "devDependencies": {
     "@napi-rs/cli": "^2.16.3",
+    "@types/chai": "^4.2.0",
+    "@types/mocha": ">=9.1.0",
```

Comment:
The way packages are structured in `pnpm` is more explicit, which means that some things that were working by coincidence no longer work (this is a good thing). One of those was that we were missing these two packages here.